BUILD.bazel

```diff
@@ -14,6 +14,9 @@ load("//:def.bzl", "gazelle", "gazelle_binary")
 gazelle(
     name = "gazelle",
     command = "fix",
+    env = {
```

Comment:
Is this accidental? Not exactly sure where a test for this would go, but probably not here.

Reply:
it was on purpose to show how it works, but I can take it off, and figure out how to write a test for it, as it's a new feature

Reply:
That would be great, let me know if you get stuck. To be honest, I'm not very familiar with Gazelle's testing infrastructure myself, but I can always ask the other maintainers.

Reply:
I think I found how to make a test that would pass, not so sure about one that would fail, we should test env variables starting with numbers, having spaces, or being empty as well as a few valid ones

Reply:
It was slightly harder than what I expected, but I finally managed to get it working, testing of rules is such a pain, but we have to do it anyway